### ISSUE TITLE
Getting chinese or japanese words from the pretrained model instead of english

### ISSUE THREAD

**Issue by strivedi02** [EmilyAlsentzer/clinicalBERT#34]
I am just trying masked word prediction on pre-trained bio_clinicalbert but instead of getting english word output I am getting Chinese or Japanese words
Below is my code:
```from transformers import BertTokenizer, BertForMaskedLM
bio_bert_tokenizer = BertTokenizer.from_pretrained('Bio_ClinicalBERT')
bio_bert_model = BertForMaskedLM.from_pretrained('Bio_ClinicalBERT').eval()
```
```
input_ids, mask_idx = encode(bio_bert_tokenizer, text_sentence)
with torch.no_grad():
    predict = bert_model(input_ids)[0]
bio_bert = decode(bio_bert_tokenizer, predict[0, mask_idx, :].topk(top_k).indices.tolist(), top_clean)
print(bio_bert)
```
And The output
![Screenshot from 2021-04-02 14-43-26](https://user-images.githubusercontent.com/8588143/113402006-fa2a9d00-93c1-11eb-8b48-cba0898d59ef.png)
I have downloaded all the model files from https://huggingface.co/emilyalsentzer/Bio_ClinicalBERT/tree/main
I don't know if am doing a naive mistake or not please excuse me as am new to this whole transformers library.

**Comment by strivedi02** [comment]
I changed the mask keyword from '[MASK]' to "\<mask\>" which resulted in this issue
